cli/src/pcluster/config/cluster_config.py

```diff
@@ -1935,7 +1944,11 @@ def __init__(self, instance_type: str, **kwargs):
 class SlurmFlexibleComputeResource(_BaseSlurmComputeResource):
     """Represents a Slurm Compute Resource with Multiple Instance Types."""
 
-    def __init__(self, instances: List[FlexibleInstanceType], **kwargs):
+    def __init__(
+        self,
+        instances: List[FlexibleInstanceType],
+        **kwargs,
+    ):
```

Comment:
Nitpick: should we revert this change, since it doesn't do anything?

Reply:
I thought about it, but then I decided to keep them.
Having the same `init style` across all resources make the code more readable.